README.md

```diff
@@ -301,6 +301,53 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Automati
 </details>
 
 
+## Debug
+<details>
+### Using Debug Log
+There are six levels of logs, which can be called explicitly or set via the ``OPENVINO_LOG_LEVEL`` environment variable:
+
+0 - ``ov::log::Level::NO``
+1 - ``ov::log::Level::ERR``
+2 - ``ov::log::Level::WARNING``
+3 - ``ov::log::Level::INFO``
+4 - ``ov::log::Level::DEBUG``
+5 - ``ov::log::Level::TRACE``
+
+When setting the environment variable OPENVINO_LOG_LEVEL > ov::log::Level::WARNING, the properties of the compiled model can be printed.
+
+For example:
+Linux
+export OPENVINO_LOG_LEVEL=3
+Windows
+set OPENVINO_LOG_LEVEL=3
+the properties of the compiled model are printed as follows:
```

Comment:
Here and above is rendered incorrectly. Please fix this

Reply:
Fixed, check, pls.